src/ProtocolFees.sol

```diff
@@ -17,18 +18,19 @@ abstract contract ProtocolFees is IProtocolFees, Owned {
     using PoolIdLibrary for PoolKey;
     using Pool for Pool.State;
     using CustomRevert for bytes4;
+    using BipsLibrary for uint256;
 
     /// @inheritdoc IProtocolFees
     mapping(Currency currency => uint256 amount) public protocolFeesAccrued;
 
     /// @inheritdoc IProtocolFees
     IProtocolFeeController public protocolFeeController;
 
-    uint256 private immutable controllerGasLimit;
+    // a percentage of the block.gaslimit denoted in basis points, used as the gas limit for fee controller calls
+    // 100 bps is 1%, at 30M gas, the limit is 300K
```

Comment:
Are we fine with this number? 

This is what we use for subscribers on periphery?

Reply:
yeah it seemed reasonable here too. 300k would be a lotttttttt. arguably id be happy to go with smaller lol